tee-worker/enclave-runtime/src/stf_task_handler.rs

```diff
@@ -55,6 +55,15 @@ pub unsafe extern "C" fn run_stf_task_handler(dpc: *const u8, dpc_size: usize) -
 			dpc.set_oneblock_notion_url(data_provider_config.oneblock_notion_url);
 			dpc.set_sora_quiz_master_id(data_provider_config.sora_quiz_master_id);
 			dpc.set_sora_quiz_attendee_id(data_provider_config.sora_quiz_attendee_id);
+			dpc.set_contest_legend_discord_role_id(
+				data_provider_config.contest_legend_discord_role_id,
+			);
+			dpc.set_contest_popularity_discord_role_id(
+				data_provider_config.contest_popularity_discord_role_id,
+			);
+			dpc.set_contest_participant_discord_role_id(
+				data_provider_config.contest_participant_discord_role_id,
+			);
```

Comment:
Are these numbers that can't be leaked?
I'm just concerned the config will get bloated soon ...

Reply:
How about we waiting a bit, until it bloating? 😂

Reply:
I must be missing something from why we took this hard-coding approach, but should these parameters come from the client? clients should be free to use these assertions more generically 🤔

Reply:
cc @higherordertech @zhouhuitian 

How is the VC designed? Are these role-ids likely to change?
If so, it should come from the client, if not (so it's more like a constant), we should hard code it instead of leaving it in a config in prod.

https://linear.app/litentry/issue/P-233/hardcode-the-data-provider-urlconstants-into-prod-enclave